clang-tools-extra/clang-doc/tool/ClangDocMain.cpp

```diff
@@ -167,7 +167,14 @@ llvm::Error getDefaultAssetFiles(const char *Argv0,
 
   llvm::SmallString<128> AssetsPath;
   AssetsPath = llvm::sys::path::parent_path(NativeClangDocPath);
-  llvm::sys::path::append(AssetsPath, "..", "share", "clang-doc");
+  llvm::sys::path::append(AssetsPath, "..");
+  llvm::SmallString<128> tempCopyDbg;
+  llvm::sys::path::native(AssetsPath, tempCopyDbg);
+  llvm::sys::path::append(tempCopyDbg, "Debug");
```

Comment:
Could this also be "Release"?

It seems like this should probably be built like the compiler resource directory is where the assets get installed into a known binary relative path.